sycl/include/CL/sycl/detail/pi.h

```diff
@@ -88,6 +88,61 @@ typedef enum {
   PI_QUEUE_INFO_REFERENCE_COUNT = CL_QUEUE_REFERENCE_COUNT
 } _pi_queue_info;
 
+typedef enum {
+  PI_IMAGE_FORMAT       = CL_IMAGE_FORMAT,
```

Comment:
please follow the convention that constants name repeat it's type name (e.g. see PI_QUEUE_INFO_DEVICE above). this would be PI_IMAGE_INFO_FORMAT